di-ipv-ticf-stub/deploy/template.yaml

```diff
@@ -126,6 +128,10 @@ Resources:
         - VPCAccessPolicy: { }
         - SSMParameterReadPolicy:
             ParameterName: stubs/core/ticf/*
+        - KMSDecryptPolicy:
+            KeyId: !Ref DynamoDBKmsKey
+        - DynamoDBCrudPolicy:
+            TableName: !Ref TicfStubUserEvidenceTable
       AutoPublishAlias: live
       Events:
         GetTicfVC:
```

Comment:
Do we need to add the management API here?

Reply:
not needed